docker/Dockerfile

```diff
@@ -32,21 +44,22 @@ RUN mkdir -p /config && \
     # Install ytdl-sub, ensure it is installed properly
     pip install --no-cache-dir ytdl_sub-*.whl && \
       ytdl-sub -h && \
+      ldconfig && \
```

Comment:
what does this do?

Reply:
while I was initially adding these changes I had some extra commands that didn't end up being needed, this line can be removed